9.2/run-postgresql.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+
+# Data dir
+export PGDATA=/var/lib/pgsql/data
+
+# Enable utf8 support. This might affect performance:
+# http://www.postgresql.org/docs/9.2/static/locale.html
+export LANG=en_US.utf8
```

Comment:
Right, will fix that tomorrow so LANG can be overridden. I might, but setting the LANG var is the simplest way to do it, I didn't see any reason to modify config file because of it. Btw, it mostly affects initdb. And yeah, the default is probably faster a bit, but without utf support, at least that's how I understand it.